File: LV.2/최대값과 최솟값/solution.py
def solution(s):
    b = s.split(" ")
    new =[]
    for i in b:
        i = int(i)
        new.append(i)

    min_value = min(new)
    min_value = str(min_value)
    max_value = max(new)
    max_value = str(max_value)


    answer = min_value+" "+max_value
    return answer

File: LV.2/최대값과 최솟값/Solution.java
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] str = s.split(" ");
        List<Integer> list = new ArrayList<Integer>();
        
        for (int i=0; i < str.length; i++) {
            int j = Integer.parseInt(str[i]);
            list.add(j);
        }
         
        String min = Integer.toString(Collections.min(list));
        String max = Integer.toString(Collections.max(list));
        
        return min + " " + max;
    }
}
